.github/workflows/simulation-1.2.yml

```
@@ -136,6 +148,10 @@ jobs:
         ./script/test get_thread_wireshark
     - name: Run
       run: |
+        ulimit -c unlimited
+        echo "$PWD/ot-core-dump/corefile-%e-%p-%t" | sudo tee /proc/sys/kernel/core_pattern
+        mkdir ot-core-dump
+        ./script/test copy_so_lib
```

```suggestion
        ./script/test prepare_coredump_upload
```

Merge all to one command?

---

We can put `echo ...` and `mkdir` into the script however we have to put `ulimit` here because `ulimit` is per-process. FYI: [Why doesn't ulimit -n work when called inside a script?](https://superuser.com/questions/1289345/why-doesnt-ulimit-n-work-when-called-inside-a-script)